generator/src/main/java/org/corfudb/generator/operations/SnapshotTxOperation.java

```diff
@@ -0,0 +1,42 @@
+package org.corfudb.generator.operations;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.generator.State;
+
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Created by box on 7/15/17.
+ */
+@Slf4j
+public class SnapshotTxOperation extends Operation {
+
+    public SnapshotTxOperation(State state) {
+        super(state);
+    }
+
+    @Override
+    public void execute() {
+        long trimMark = state.getTrimMark();
+        Random rand = new Random();
+        long delta = rand.nextInt(10) + 1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this addition operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)